cli/plasmo/src/commands/package.ts

```diff
@@ -18,7 +19,7 @@ async function packageCmd() {
 
   const plasmoManifest = await createManifest(bundleConfig)
 
-  await zipBundle(plasmoManifest.commonPath)
+  await zipBundle(plasmoManifest.commonPath, hasFlag("--with-source-maps"))
```

Comment:
[nit] do we need to document this --with-source-maps flag anywhere?

I realized plasmo package --help doesn't show anything.

Reply:
Yeah will add this as a docs ticket for batch docs update in the future!

Reply:
https://github.com/orgs/PlasmoHQ/projects/2